src/app/core/env.ts

```diff
@@ -1,4 +1,12 @@
 import {app as electronApp} from "electron"
+import * as brimPackage from "../../../package.json"
+
+export const toTaggedZedRepoUrl = (path: string, isDirectory = false) => {
```

Comment:
Wasn't quite sure where the best logical home was for this util function; open to suggestions if you disagree with core/env

Reply:
How about a new file called `app/core/links` ?